src/MICore/CommandFactories/MICommandFactory.cs

```diff
@@ -609,6 +610,18 @@ internal string EscapeQuotes(string str)
             return str.Replace("\"", "\\\"");
         }
 
+        internal string HandleInvalidChars(string str)
+        {
+            char[] invalidChars = { '\r', '\n' };
```

Comment:
Do we also need to remove `\0`? Or is that an error in your description?

Reply:
Not following you. Where did I mention \0? Or are you misreading () as 0?